src/main/java/org/sunbird/common/model/SunbirdApiHierarchyResultContent.java

```diff
@@ -22,6 +22,8 @@
 	private String streamingUrl;
 	private List<SunbirdApiHierarchyResultContent> children;
 	private List<SunbirdApiBatchResp> batches;
+	private List<String> leafNodes;
```

Comment:
We don't need to store the leafNodes details from hierarchyReponse object. 
leafNodes are child resource details for the particular course - which is not required.
Also - this not used anywhere else.